pimd/pim_igmpv3.c

```diff
@@ -1826,6 +1827,64 @@ void igmp_v3_recv_query(struct gm_sock *igmp, const char *from_str,
 	} /* s_flag is clear: timer updates */
 }
 
+static bool igmp_pkt_grp_addr_ok(struct interface *ifp, const char *from_str,
+				 struct in_addr grp, int rec_type)
+{
+	struct pim_interface *pim_ifp;
+	struct in_addr grp_addr;
+
+	pim_ifp = ifp->info;
+
+	/* determine filtering status for group */
+	if (pim_is_group_filtered(pim_ifp, &grp)) {
+		if (PIM_DEBUG_IGMP_PACKETS) {
+			zlog_debug(
+				"Filtering IGMPv3 group record %pI4 from %s on %s per prefix-list %s",
+				&grp.s_addr, from_str, ifp->name,
+				pim_ifp->boundary_oil_plist);
+		}
+		return false;
+	}
+
+	/*
+	 * If we receive a igmp report with the group in 224.0.0.0/24
+	 * then we should ignore it
+	 */
+
+	grp_addr.s_addr = ntohl(grp.s_addr);
+
+	if (pim_is_group_224_4(grp_addr)) {
```

Comment:
224/4 prefix match covers all multicast addresses including SSM address. So, effectively we ignore learning IGMPv3 reports here.

Reply:
Fixed